libs/block-editor/src/lib/plugins/bubble-link-form.plugin.ts

```diff
@@ -120,7 +123,7 @@ export class BubbleLinkFormView {
         // Afther show the component set values
         this.setInputValues();
         this.focusInput();
-        this.setTippyPosition();
+        this.tippy?.setProps({ getReferenceClientRect: () => this.setTippyPosition() });
```

Comment:
What's the diff?

Reply:
Tippy automatically triggers the `getReferenceClientRect ` every time it needs to update the position. That is why it is recommended to pass a function that returns `ClientRect` object to that property

Reply:
Tippy uses this property `getReferenceClientRect` to set the position of the tippy every time it is necessary, because of that they recommend that we pass it a function whose return is `RectClient`.

[More information](https://atomiks.github.io/tippyjs/v6/all-props/#getreferenceclientrect)